packages/calypso-e2e/src/browser-helper.ts

```diff
@@ -128,3 +122,19 @@ export async function getDefaultLoggerConfiguration(): Promise< LaunchOptions >
 		},
 	};
 }
+
+/**
+ * Returns the artifact directory where logs, screenshots and video recordings are stored.
+ *
+ * @returns {Promise<string>} Path to the artifdact directory.
+ */
+export async function getArtifactDir(): Promise< string > {
+	const { testPath } = getState() as { testPath: string };
+	const sanitizedTestFilename = path.basename( testPath, path.extname( testPath ) );
+	const resultsPath = path.join( process.cwd(), 'results' );
+	await mkdir( resultsPath, { recursive: true } );
+	const out = await mkdtemp( path.join( resultsPath, sanitizedTestFilename + '-' ) );
+	console.log( out );
+	// return await mkdtemp( path.join( resultsPath, sanitizedTestFilename + '-' ) );
+	return out;
```

Comment:
This looks like debugging code left over? Otherwise this works great!

Reply:
Yep, I noticed now - will remove. I was in a bit of a rush to get lunch.